aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/ReadHandler.java

```diff
@@ -40,7 +40,7 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                 .handleError((describeRequest, exception, client, resourceModel, ctx) -> Commons.handleException(
                         ProgressEvent.progress(resourceModel, ctx),
                         exception,
-                        DEFAULT_CUSTOM_DB_ENGINE_VERSION_ERROR_RULE_SET))
+                        DEFAULT_CUSTOM_DB_ENGINE_VERSION_ERROR_RULE_SET, requestLogger))
```

Comment:
nit: format this so that each parameter to be in single line. Male sure you fix all occurrence of it.
```suggestion
                        DEFAULT_CUSTOM_DB_ENGINE_VERSION_ERROR_RULE_SET,
                        requestLogger))
```